make/Images.gmk

```diff
@@ -180,6 +185,14 @@ ifeq ($(BUILD_CDS_ARCHIVE), true)
     $(foreach v, $(JVM_VARIANTS), \
       $(eval $(call CreateCDSArchive,$v,_nocoops)) \
     )
+    ifeq ($(BUILD_CDS_ARCHIVE_COH), true)
+      $(foreach v, $(JVM_VARIANTS), \
+        $(eval $(call CreateCDSArchive,$v,_coh)) \
+      )
+      $(foreach v, $(JVM_VARIANTS), \
+        $(eval $(call CreateCDSArchive,$v,_nocoops_coh)) \
+      )
+    endif
   endif
```

Comment:
It is necessary for some of the tests. I guess I could make it a prerequisite, but that would not work for the mainline version of this patch, since that one still lacks UseCOH. What would be the advantage of making this a separate change?

Reply:
Okay, I'll split out the COH archive changes.